mobile-app/app/api/cache/index.ts

```diff
@@ -0,0 +1,34 @@
+import Cache from 'lru-cache'
+
+class LruCache {
+  private readonly cache
+
+  constructor () {
+    // setting maxAge to 10 min
+    this.cache = new Cache({
```

Comment:
Add some options:
1. Max = to ensure we don't populate memory especially for older devices

Reply:
setting max to `max: 10000,`